src/globus_cli/commands/collection/create/guest.py

```diff
@@ -0,0 +1,221 @@
+
+from globus_cli.commands.collection._common import (
+    filter_fields,
+    standard_collection_fields,
+)
+from globus_cli.constants import ExplicitNullType
+from globus_cli.endpointish import EntityType
+from globus_cli.login_manager import (
+    LoginManager,
+    MissingLoginError,
+    read_well_known_config,
+)
+from globus_cli.login_manager.context import LoginContext
+from globus_cli.parsing import command, endpointish_params, mutex_option_group
+from globus_cli.services.gcs import CustomGCSClient
+from globus_cli.termio import TextMode, display
+
+
+@command("guest", short_help="Create a GCSv5 Guest Collection")
+@click.argument("MAPPED_COLLECTION_ID", type=click.UUID)
+@click.argument("COLLECTION_BASE_PATH", type=str)
+@click.option(
+    "--user-credential-id",
+    type=click.UUID,
+    default=None,
+    help="ID identifying a registered local user to associate with the new collection",
+)
+@click.option(
+    "--local-username",
+    type=str,
+    default=None,
+    help=(
+        "[Alternative to --user-credential-id] Local username to associate with the new"
+        " collection (must match exactly one pre-registered User Credential ID)"
+    ),
+)
+@mutex_option_group("--user-credential-id", "--local-username")
+@endpointish_params.create(name="collection")
+@click.option(
+    "--identity-id",
+    default=None,
+    help="User who should own the collection (defaults to the current user)",
+)
+@click.option(
+    "--public/--private",
+    "public",
+    default=True,
+    help="Set the collection to be public or private",
+)
+@click.option(
+    "--enable-https/--disable-https",
+    "enable_https",
+    default=None,
+    help=(
+        "Explicitly enable or disable  HTTPS support (requires a managed endpoint "
+        "with API v1.1.0)"
+    ),
+)
+@LoginManager.requires_login("auth", "transfer")
+def collection_create_guest(
+    login_manager: LoginManager,
+    *,
+    mapped_collection_id: uuid.UUID,
+    collection_base_path: str,
+    user_credential_id: uuid.UUID | None,
+    local_username: str | None,
+    contact_info: str | None | ExplicitNullType,
+    contact_email: str | None | ExplicitNullType,
+    default_directory: str | None | ExplicitNullType,
+    department: str | None | ExplicitNullType,
+    description: str | None | ExplicitNullType,
+    display_name: str,
+    enable_https: bool | None,
+    force_encryption: bool | None,
+    identity_id: str | None,
+    info_link: str | None | ExplicitNullType,
+    keywords: list[str] | None,
+    public: bool,
+    organization: str | None | ExplicitNullType,
+    user_message: str | None | ExplicitNullType,
+    user_message_link: str | None | ExplicitNullType,
+    verify: dict[str, bool],
+) -> None:
+    """
+    Create a GCSv5 Guest Collection.
+
+    Create a new guest collection, named DISPLAY_NAME, as a child of
+    MAPPED_COLLECTION_ID. This new guest collection's file system must be rooted at
```

Comment:
Very small, but I think important, change here:
```suggestion
    MAPPED_COLLECTION_ID. This new guest collection's file system will be rooted at
```

It's not "must be", as in "this is a constraint you must meet in order to use this command". It's "will be", as in "this is what will happen when you use this command".